crates/ty_python_semantic/resources/corpus/cyclic_symbol_in_comprehension.py

```diff
@@ -1,13 +1,7 @@
-# Documentation of two fuzzer panics involving comprehensions
+# Regression test for https://github.com/astral-sh/ruff/pull/20962
```

Comment:
Can you say more about why you converted this file to a corpus test? It's not clear to me why that would be preferred or required.